website/frontend_types.py

```diff
@@ -143,7 +143,8 @@ def get(self, key, default=None):
 
     @staticmethod
     def from_teacher_adventure_database_row(row):
-        text, example_code = halve_adventure_content(row["content"])
+        text, example_code = halve_adventure_content(
+            row["formatted_content"] if row.get("formatted_content") else row["content"])
```

Comment:
this work but did you know that you can supply get with a default argument too? This code does the same:

`row.get("formatted_content", row["content"])`

(See: https://docs.python.org/3/library/stdtypes.html#dict.get)

Reply:
Sure, i use this elsewhere as well, this commit i rolled back from another pr. WIll change it now.